snippet-extractor-output/snippets.yaml

```diff
@@ -3065,6 +3065,52 @@
         directDependents.areas.items[i].format.fill.color = "Yellow";
         console.log(`  ${directDependents.areas.items[i].address}`);
       }
+      await context.sync();
+    });
+'Excel.Range#getExtendedRange:member(1)':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/excel/85-preview-apis/range-get-range-edge.yaml
+
+    await Excel.run(async (context) => {
+      // Get the selected range.
+      const range = context.workbook.getSelectedRange();
+
+      // Get the active cell in the workbook.
+      const activeCell = context.workbook.getActiveCell();
+
+      // Get all the cells from the currently selected range to the right-most edge of the used range.
+      // This method acts like the Ctrl+Shift+Arrow key keyboard shortcut while a range is selected.
+      const extendedRange = range.getExtendedRange(
+        "Right", // Specify the direction as a string.
+        activeCell // If the selected range contains more than one cell, the active cell must be defined.
+      );
+      extendedRange.select();
+
+      await context.sync();
+    });
+  - >-
```

Comment:
Are you sure you want both these samples mapping to the same block in the ref docs? I think just one of them for getExtendedRange should be sufficient.

Reply:
I included both because they show different ways to specify the direction.

Reply:
Let's see how it looks in the ref docs tomorrow. I don't think either way is bad.